tests/test-bulk-usermeta-endpoints.ts

```diff
@@ -722,6 +728,39 @@ describe('GET /multi/metadata/:userIds/:keys', () => {
                     done();
                 });
         });
+
+        it('does not use multiple get request with single user', done => {
+            // Fake usermeta response
+            server.get('/usermeta/v1/auth/:authToken/:keys', (req, res, next) => {
+                console.log(req.params.authToken, req.params.keys);
```

Comment:
Do you want to make sure the get request was only called once? (this is what the description of the test says).

You can increment a counter at each invocation, expect it to be always "1".

Reply:
... once this is done, you can merge the PR, it looks ok.